src/Compatibility/Core/src/Windows/WindowsBasePage.cs

```diff
@@ -1,83 +1,83 @@
-using System;
-using System.ComponentModel;
-using Windows.ApplicationModel;
+//using System;
+//using System.ComponentModel;
+//using Windows.ApplicationModel;
 
-namespace Microsoft.Maui.Controls.Compatibility.Platform.UWP
-{
-	public abstract class WindowsBasePage : Microsoft.UI.Xaml.Window
-	{
+//namespace Microsoft.Maui.Controls.Compatibility.Platform.UWP
+//{
+//	public abstract class WindowsBasePage : Microsoft.UI.Xaml.Window
+//	{
 
-		Application _application;
+//		Application _application;
 
-		public WindowsBasePage()
-		{
-			if (!Windows.ApplicationModel.DesignMode.DesignModeEnabled)
-			{
+//		public WindowsBasePage()
```

Comment:
I'm just commenting this code out but not deleting it so that we have it for reference to make sure we've replaced all necessary parts of WindowsBasePage